JS/Lesson26/onur/editPost/edit.js

```diff
@@ -0,0 +1,35 @@
+
+
+const urlParams = new URLSearchParams(window.location.search);
+const titleParam = urlParams.get('title');
```

Comment:
Title and body of the post are a lot of data to share through query parameters. It's better to only share small pieces of data like id of the post to make urls easier to work with. When you have an id of the post, you can fetch the post data on the edit post page. Using 'onload' even listener in this case would be the right time to fetch the post data.